src/main/java/com/bigchaindb/util/NetworkUtils.java

```diff
@@ -42,14 +53,16 @@ public static void sendPostRequest(String url, RequestBody body,
         BigChainDBGlobals.getHttpClient().newCall(request).enqueue(new Callback() {
```

Comment:
The timeout should be passed to the request function as in the [Python driver](https://github.com/bigchaindb/bigchaindb-driver/blob/master/bigchaindb_driver/connection.py#L86)  or a timeout function should be implemented as in the [JS-driver](https://github.com/bigchaindb/js-bigchaindb-driver/blob/master/src/baseRequest.js#L26) . This is needed in case the request takes more time to respond that the timeout specified by the user

Reply:
Timeout is passed to HttpClient already when we build http client in `buildDefaultHttpClient()` method in `BigchainDbConfigBuilder.java`. That is the timeout that is used to make request which is done by http client that we created in `buildDefaultHttpClient()` method. (`BigChainDBGlobals.getHttpClient().newCall(request))`. It can be tested manually by using that timer of the phone (any device) and then shutting down all nodes. Then the driver will throw `TimeoutException` as soon as given timeout is reached.

Reply:
Okay I see. Then everything looks good, I think you can merge. :+1: